knowledge/src/main/java/com/google/android/fhir/knowledge/KnowledgeManager.kt

```diff
@@ -46,9 +43,20 @@ import timber.log.Timber
  * individually as JSON files or from FHIR NPM packages.
  *
  * Coordinates the management of knowledge artifacts by using the three following components:
- * - database: indexing knowledge artifacts stored in the local file system,
- * - file manager: managing files containing the knowledge artifacts, and
- * - NPM downloader: downloading from an NPM package server the knowledge artifacts.
+ * - knowledgeDatabase: indexing knowledge artifacts stored in the local file system,
+ * - npmFileManager: managing files containing the knowledge artifacts, and
+ * - npmPackageDownloader: downloading from an NPM package server the knowledge artifacts.
```

Comment:
```suggestion
 * - npmPackageDownloader: downloading the knowledge artifacts from an NPM package server .
```
a bit simpler to read?